Assets/Scripts/EnemySystem/Enemy Child Scripts/LungeEnemy.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ILOVEYOU
+{
+    namespace EnemySystem
+    {
+        public class LungeEnemy : Enemy
+        {
+            private float m_lungeCooldown = 5f;
```

Comment:
Would we want these accessible in inspector?

Reply:
ooh yeah would probably be good I just did that for testing lol

Reply:
can you edit this on github?

Reply:
maybe hang on